docs/3.0.0-beta.x/guides/deployment.md

```diff
@@ -65,1382 +65,18 @@ We highly recommend using [pm2](https://github.com/Unitech/pm2/) to manage your
-            }
-            console.log(`stdout: ${stdout}`);
-            console.log(`stderr: ${stderr}`);
-          }
-        );
-      }
-    });
-
-    res.end();
-  })
-  .listen(8080);
-```
-
-- Allow the port to communicate with outside web traffic for `port 8080`:
-
-```bash
-sudo ufw allow 8080/tcp
-sudo ufw enable
-
-Command may disrupt existing ssh connections. Proceed with operation (y|n)? y
-Firewall is active and enabled on system startup
-```
-
-Earlier you setup `pm2` to start the services (your **Strapi project**) whenever the **Droplet** reboots or is started. You will now do the same for the `webhook` script.
-
-- Install the webhook as a `Systemd` service
-
-  - Run `echo $PATH` and copy the output for use in the next step.
-
-```
-echo $PATH
-
-/home/your-name/.npm-global/bin:/home/your-name/bin:/home/your-name/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin
-```
-
-- Create a `webhook.service` file:
-
-```bash
-cd ~
-sudo nano /etc/systemd/system/webhook.service
-```
-
-- In the `nano` editor, copy/paste the following script, but make sure to replace `your-name` **in two places** with your username. Earlier, you ran `echo $PATH`, copy this to the `Environment=PATH=` variable, then save and exit:
-
-```bash
-[Unit]
-Description=Github webhook
-After=network.target
-
-[Service]
-Environment=PATH=your_path
-Type=simple
-User=your-name
-ExecStart=/usr/bin/nodejs /home/your-name/NodeWebHooks/webhook.js
-Restart=on-failure
-
-[Install]
-WantedBy=multi-user.target
-```
-
-- Enable and start the new service so it starts when the system boots:
-
-```bash
-sudo systemctl enable webhook.service
-sudo systemctl start webhook
-```
-
-- Check the status of the webhook:
-
-```bash
-sudo systemctl status webhook
-```
-
-- You may test your **webhook** by following the instructions [here](https://www.digitalocean.com/community/tutorials/how-to-use-node-js-and-github-webhooks-to-keep-remote-projects-in-sync#step-4-testing-the-webhook).
-
-### Further steps to take
-
-- You can **add a domain name** or **use a subdomain name** for your Strapi project, you will need to [install NGINX and configure it](https://www.digitalocean.com/community/tutorials/how-to-install-nginx-on-ubuntu-18-04).
-- Deny traffic to Port 1337. You have set-up a proxy using Nginx, you now need to block access by running the following command:
-
-```
-cd ~
-sudo ufw deny 1337
-```
-
-- To **install SSL**, you will need to [install and run Certbot by Let's Encrypt](https://www.digitalocean.com/community/tutorials/how-to-secure-nginx-with-let-s-encrypt-on-ubuntu-18-04).
-- Set-up [Nginx with HTTP/2 Support](https://www.digitalocean.com/community/tutorials/how-to-set-up-nginx-with-http-2-support-on-ubuntu-18-04) for Ubuntu 18.04.
+## Digital Ocean
 
-Your `Strapi` project has been installed on a **Digital Ocean Droplet** using **Ubuntu 18.04**.
+To see the documentation go [there](./deployment/digital-ocean.md)
```

Comment:
Is it just a change of architecture? Why?

Reply:
We will slowly continue to change parts of the architecture to make navigating the docs clearer. And to start using `includes` to stop repeating reusable blocks of code or text. 

This particular change will make more sense as we add more Deployment options.